db/seeds.rb

```diff
@@ -1,7 +1,15 @@
+require 'json'
+
 Doorkeeper::Application.create!(name: 'Web', superapp: true, redirect_uri: Doorkeeper.configuration.native_redirect_uri, scopes: 'read write follow')
 
 if Rails.env.development?
   domain = ENV['LOCAL_DOMAIN'] || Rails.configuration.x.local_domain
   admin  = Account.where(username: 'admin').first_or_create!(username: 'admin')
   User.where(email: "admin@#{domain}").first_or_initialize(email: "admin@#{domain}", password: 'mastodonadmin', password_confirmation: 'mastodonadmin', confirmed_at: Time.now.utc, admin: true, account: admin).save!
 end
+
+reserved_words = JSON.parse(File.read('db/reserved_username.json'))
+
+reserved_words.each do |word|
+  ReservedUsernameList.create(word: word)
```

Comment:
Putting this in seeds is confusing, because it won't update the database if words are added or removed from the json file. I think this probably shouldn't be in the database—instead, it should probably be loaded from the filesystem when the app loads.